tutorials/printing.adoc

```diff
@@ -20,11 +20,11 @@ TODO: Krátká specifikace tiskárny?
 Na fotografii níže se seznamte s pojmy, které budou dále ve studijním textu zmíněny.
 
 .Tiskána RebeliX
-image::../images/rebelix-photo-annotated.jpg[Rebelix,600]
-<1> Tisková deska
+image::../images/rebelix-photo-annotated.jpg[Rebelix, width=720, link=../images/rebelix-photo-annotated.jpg]
```

Comment:
až teď jsem si všiml. většina cvik má nenapracované obrázky. Tady bych to dodržel tak,y ať nám nevzniká v images/ bordel.